src/art/DhanushNehru/index.html

```diff
@@ -0,0 +1,8 @@
+<html>
+<head>
+</head>
+<body onload="drawChessboard()">
+  <center><canvas id="chessboard" width="700px" height="400px"></canvas></center>
```

Comment:
`center` HTML element is deprecated. Can this be changed please?

Reply:
Yeah, I removed it